portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -76,4 +87,22 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     // Redirect back to the HTML page.
     response.sendRedirect("/index.html");
   }
+
+/** Returns the choice entered by the player, or -1 if the choice was invalid. */
+  private int getNumComments(HttpServletRequest request) {
+    // Get the input from the form.
```

Comment:
Usually unnecessary to document "what" code is doing unless what's being done is particularly complex.

Reply:
I tried to improve my comments so they are more readable and less about what is happening in my code. Let me know if I can make more changes to make them better!